ubuntu-advantage-client/ec2_cleanup.py

```diff
@@ -19,62 +19,125 @@ def parse_args():
         help=("Tag used to filter cloud resources for deletion. "
               "Default: {}".format(CI_DEFAULT_TAG))
     )
+    parser.add_argument(
+        "-o", "--older-than", dest="older_than", action="store",
```

Comment:
"older-than" I think of a number of days or something. e.g. older than 30 days or older than 1yr. What about hinting you're looking for a date with "before-date" and then removing on that? 

I also think the hh:mm:ss is noise in the sense of an automated cleanup script. Can we just keep the date and leave the time to be midnight of that date?